examples/tv-app/tv-common/include/AppTv.h

```diff
@@ -139,14 +139,10 @@ class DLL_EXPORT ContentAppFactoryImpl : public ContentAppFactory
 
     void AddAdminVendorId(uint16_t vendorId);
 
-protected:
-    ContentAppImpl mContentApps[APP_LIBRARY_SIZE] = {
-        ContentAppImpl("Vendor1", 1, "exampleid", 11, "Version1", "34567890"),
-        ContentAppImpl("Vendor2", 65521, "exampleString", 32768, "Version2", "20202021"),
-        ContentAppImpl("Vendor3", 9050, "App3", 22, "Version3", "20202021"),
-        ContentAppImpl("TestSuiteVendor", 1111, "applicationId", 22, "v2", "20202021")
-    };
+    void AddContentApp(uint16_t vendorId, uint16_t productId);
 
+protected:
+    std::vector<std::unique_ptr<ContentAppImpl>> mContentApps;
```

Comment:
Do you think it would be good to have a single app by default which provides the setup code for the default vid of the linux casting app, and then provide an easy way to change the vid of the the linux casting app?

This would keep fewer commissioning steps for the developer, but still allow them to test the uninstalled app case.

Reply:
Sure I can add it. 

1. Will this single content app work with other content apps or will it replace the existing content apps?
2. If it should work with existing content apps, I guess it should then be returned as a member of mContentApps even though it's not?

Reply:
Added `app uninstall` command that can remove the app.